src/api/BdmsContextExtensions.cs

```diff
@@ -0,0 +1,360 @@
+﻿using BDMS.Models;
+using Bogus;
+using Microsoft.EntityFrameworkCore;
+using NetTopologySuite.Geometries;
+using System.Globalization;
+
+namespace BDMS;
+
+#pragma warning disable CA1505
+/// <summary>
+/// The EF database context containing data for the BDMS application.
+/// </summary>
+public static class BdmsContextExtensions
+{
+    /// <summary>
+    /// Seed data for <see cref="Workgroup"/>, <see cref="Borehole"/> and <see cref="Stratigraphy"/>.
+    /// </summary>
+    public static void SeedData(this BdmsContext context)
+    {
+        using var transaction = context.Database.BeginTransaction();
+
+        // Set Bogus Data System Clock
+        Bogus.DataSets.Date.SystemClock = () => DateTime.Parse("01.01.2022 00:00:00", new CultureInfo("de_CH", false));
+
+        // Seed Workgroups
+        var workgroup_ids = 2;
+        var workgroupRange = Enumerable.Range(workgroup_ids, 5);
+        var fakeWorkgroups = new Faker<Workgroup>()
+           .StrictMode(true)
+           .RuleFor(o => o.Id, f => workgroup_ids++)
+           .RuleFor(o => o.Name, f => f.Music.Genre())
+           .RuleFor(o => o.Created, f => f.Date.Past().OrNull(f, .1f))
+           .RuleFor(o => o.Disabled, f => f.Date.Past().OrNull(f, .1f))
+           .RuleFor(o => o.IsSupplier, f => f.Random.Bool().OrNull(f, .1f))
+           .RuleFor(o => o.Settings, f => null)
+           .RuleFor(o => o.Boreholes, _ => default!);
+        Workgroup SeededWorkgroups(int seed) => fakeWorkgroups.UseSeed(seed).Generate();
+        context.Workgroups.AddRange(workgroupRange.Select(SeededWorkgroups));
+        context.SaveChanges();
+
+        // ranges for exsiting tables
+        var userRange = Enumerable.Range(1, 5);
+
+        var cantonRange = Enumerable.Range(1, 51);
+        var municipalitieRange = Enumerable.Range(1, 2371);
+
+        // Seed Boreholes
+        var borehole_ids = 1000;
```

Comment:
Q: Wollen wir von Anfang an ein bisschen mehr Daten in die DB seeden, z.B. 3'000?

Reply:
Wenn ich es schaffe das seeden noch etwas zu optimieren können wir es probieren. 3000 boreholes heisst dann ca. 15'000 stratigraphien und 150'000 Layer.